python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1088,6 +1093,187 @@ def run(self, handle, funcs=None, dataId=None, band=None, **kwargs):
+        default="{band}_{model}FluxErr",
+        check=lambda x: ("{band}" in x) and ("{model}" in x,),
+    )
+    model_flux_name = pexConfig.Field[str](
+        doc="The extended object model to use to compared to PSF model fluxes",
+        default="sersic",
+    )
+    output_column = pexConfig.Field[str](
+        doc="Name of the output column. Must contain the {band} template",
+        default="{band}_model_extendedness",
+        check=lambda x: "{band}" in x,
+    )
+    psf_column_flux = pexConfig.Field[str](
+        doc="The name of the PSF flux column. Must contain the {band} template.",
+        default="{band}_psfFlux",
+        check=lambda x: "{band}" in x,
+    )
+    psf_column_flux_err = pexConfig.Field[str](
+        doc="The name of the PSF flux error column. Must contain the {band} template.",
+        default="{band}_psfFluxErr",
+        check=lambda x: "{band}" in x,
+    )
+    size_column = pexConfig.Field[str](
+        doc="The column to use for applying size cuts. Must contain the {axis} template.",
+        default="exponential_reff_{axis}",
+    )
+
+    def getInputSchema(self) -> Iterable[tuple[str, type[NDArray]]]:
+        size_column = self.size_column
+        schema = [
+            (size_column.format(axis=axis), NDArray[float]) for axis in ("x", "y")
+        ]
+        model = self.model_flux_name
+        for column in (
+                self.psf_column_flux, self.psf_column_flux_err,
+                self.model_column_flux, self.model_column_flux_err,
+        ):
+            schema.extend([
+                (column.format(band=band, model=model), NDArray[float]) for band in self.bands
+            ])
+
+        return schema
+
+    def validate(self):
+        super().validate()
+        errors = []
+        for name, band_combined in self.bands_combined.items():
+            bands = band_combined.split(",")
+            bands_missing = [band for band in bands if band not in self.bands]
+            if bands_missing:
+                errors.append(
+                    f"self.bands_combined[{name}] contains bands={bands_missing} not in {self.bands=}"
+                )
+        if errors:
+            raise ValueError(f"Validation failed due to errors: {'; '.join(errors)}")
+
+    def _get_fluxes(self, table, band: str):
+        model = self.model_flux_name
+        flux_psf, fluxerr_psf, flux_model, fluxerr_model = (
+            np.array(table[column.format(band=band, model=model)])
+            for column in (
+                self.psf_column_flux, self.psf_column_flux_err,
+                self.model_column_flux, self.model_column_flux_err,
+            )
+        )
+        return flux_psf, fluxerr_psf, flux_model, fluxerr_model
+
+
+class ModelExtendednessColumnAction(ExtendednessColumnActionBase):
+    fluxerr_coefficent = pexConfig.Field[float](
+        doc="The coefficient to multiply the flux error by when adding to the model flux.",
+        default=1.0,
```

Comment:
My read from the plots was that you had settled on 0.5 for this instead of 1.0. My vote is still zero but I could live with 0.5. I think it's ok to merge the code and backport a tweak to this value later if you want to get a third opinion etc.

Reply:
I set it to 0.5, partly because it's slightly better and because I think it's worth discussing in the paper.